include/extractor/profile_properties.hpp

```diff
@@ -11,8 +11,8 @@ namespace extractor
 struct ProfileProperties
 {
     ProfileProperties()
-        : traffic_signal_penalty(0), u_turn_penalty(0), continue_straight_at_waypoint(true),
-          use_turn_restrictions(false), left_hand_driving(false)
+        : traffic_signal_penalty(0), u_turn_penalty(0), max_speed_for_map_matching(180/3.6),
```

Comment:
Maybe add a 

    const constexpr auto DEFAULT_MAX_SPEED = ..;

at the top of the struct and then use it here